tests/avalon_test_framework/src/libs/verification_libs.py

```diff
@@ -221,7 +222,7 @@ def check_worker_create_receipt_response(response):
     """
     This function will verifies the create receipt response
     """
-    if env.blockchain_type == "ethereum":
+    if blockchain_type == "ethereum":
```

Comment:
From where are you getting the blockchain_type? shouldn't it be env.blockchain_type or now env['blockchain_type'] ?

Reply:
Changes addressed